freestyle/shared/src/main/scala/freestyle/package.scala

```diff
@@ -58,15 +58,14 @@ package object freestyle {
     def liftPar[F[_], A](freeap: FreeS.Par[F, A]): FreeS[F, A] =
       Free.liftF(freeap)
 
-    def inject[F[_], G[_]]: FreeSParInjectPartiallyApplied[F, G] =
+    def inject[F[_], G[_]](implicit I: Inject[F, G]): FreeSParInjectPartiallyApplied[F, G] =
```

Comment:
If you pass the implicit `Inject` instance here, I don't think we need `FreeSParInjectPartiallyApplied`.

Instead it could probably be:
```scala
def inject[F[_], G[_]](implicit I: Inject[F, G]): F ~> FreeS.Par[G, ?] =
  λ[F ~> FreeS.Par[G, ?]](fa => FreeApplicative.lift(I.inj(fa))
```